system-test/datastore.ts

```diff
@@ -308,6 +308,39 @@ describe('Datastore', () => {
       await datastore.delete(postKey);
     });
 
+    // TODO: Modify this
+    it('should save/get/delete from a snapshot', async () => {
+      function sleep(ms: number) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+      }
+      const post2 = {
+        title: 'Another way to make pizza',
+        tags: ['pizza', 'grill'],
+        publishedAt: new Date(),
+        author: 'Silvano',
+        isDraft: false,
+        wordCount: 400,
+        rating: 5.0,
+        likes: null,
+        metadata: {
+          views: 100,
+        },
+      };
+      const path = ['Post', 'post1'];
+      const postKey = datastore.key(path);
```

Comment:
nit - I would rename this to `postKey1` or something similar to reference this is related to post1 rather than post2

Reply:
The same `postKey` is used for both `post1` and `post2`. There is only one `postKey`.